### DISCUSSION TITLE
Noble Usdc

### DISCUSSION THREAD

**Discussion by 31trainman** [akash-network/community#451] [Economics]
Has the akash team looked into adding support for noble networks Usdc?

Adding support for noble Usdc makes it easier for users to get deployments running on Akash Network and is easier to get then say Usdc on axelar network. Also because it's a stablecoin its value doesn't fluctuate as much as the akt token does. Plus you can get noble network Usdc on Coinbase or through a circle business banking account.

**Comment by rodri-r** [comment]
I think this is a must
